src/actions/sendCallLogs.js

```diff
@@ -0,0 +1,23 @@
+export default async (roomName, log) => {
+
+    const logWithoutAction = () => {
+        delete log.action
+        return log
+    }
```

Comment:
You don't need a function. Do directly delete log.action

Reply:
But I need log.action as the value of roomLogInfo.action (roomLogInfo was required to be structured in this way for backend needs). Then, log (without action property to avoid redundancy) is required to be the value of roomLogInfo.event

Reply:
you can use destructuring
```
const {action, ...event} = log
```
You don't mutate the original object with this approach. 

```
 const roomLogInfo = {
        room: roomName,
        action,
        event
    }
```

Reply:
Now that's neat. Thank you! Done